test/cases/expressions/selection-brace.hack

```diff
@@ -0,0 +1,5 @@
+$dynamic_item->{$primary_key};
+
+$dynamic_item->{$primary_key}->{$primary_key2}->{$primary_key3};
+
+$dynamic_item->{$primary_key}->$primary_key2->{$primary_key3};
```

Comment:
Whoaaaa what didn't know this was valid Hack

Reply:
Ooooh it allows you to access class variables dynamically. Cool, didn't know about this.